src/e2e-test/features/bigquery/sink/BigQueryToBigQuerySink.feature

```diff
@@ -195,3 +195,141 @@ Feature: BigQuery sink - Verification of BigQuery to BigQuery successful data tr
     Then Open and capture logs
     Then Verify the pipeline status is "Succeeded"
     Then Verify the partition table is created with partitioned on field "bqPartitionFieldTime"
+
+  @BQ_UPSERT_SOURCE_TEST @BQ_UPSERT_SINK_TEST @EXISTING_BQ_CONNECTION
+  Scenario:Validate successful records transfer from BigQuery to BigQuery with Advanced operations Upsert with updating the destination table schema and destination table must exist.
```

Comment:
Change to --- Validate successful records transfer from BigQuery source to BigQuery sink with Upsert operation by updating destination table schema and destination table exists with records in it.

Reply:
changed